src/experimental/java/org/quiltmc/loader/impl/metadata/qmj/FabricModMetadataWrapper.java

```diff
@@ -115,6 +115,11 @@ public String description() {
 		return breaks;
 	}
 
+	@Override
+	public ModLoadType loadType() {
+		return ModLoadType.IF_POSSIBLE;
```

Comment:
shouldn't this be ALWAYS? that is the behavior normally expected in fabric

Reply:
Nope, fabric's old mod loading behaviour was definitely `IF_POSSIBLE` for jar-in-jars, it's just that it works the same way as `ALWAYS` most of the time so people assumed that was how it works.

For example if you just jar-in-jar a library, but then conflict with it, it won't get loaded - however if it defaulted to `ALWAYS` then it would crash.

Reply:
But this is a FabricModMetadataWrapper, *not* a JIJ wrapper? Unless the name for this class is awful/

Reply:
Oh right people don't tend to like my names :p

`FabricModMetadataWrapper` is a wrapper which converts a fabric metadata object into a quilt metadata object. So, this is effectively:

```java
public final class FabricModMetadataWrapper implements QuiltModMetadata {
  private final FabricModMetadata fabricMeta;
}
```